rules/linux/execution_shell_via_child_tcp_utility_linux.toml

```diff
@@ -0,0 +1,102 @@
+[metadata]
+creation_date = "2023/11/02"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+This detection rule identifies suspicious network traffic patterns associated with TCP reverse shell activity. This
+activity consists of a network event that is followed by the creation of a shell process with suspicious command line
+arguments. An attacker may establish a Linux TCP reverse shell to gain remote access to a target system.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Reverse Shell via Child"
+references = [
+    "https://github.com/swisskyrepo/PayloadsAllTheThings/blob/master/Methodology%20and%20Resources/Reverse%20Shell%20Cheatsheet.md"
+]
+risk_score = 47
+rule_id = "2138bb70-5a5e-42fd-be5e-b38edf6a6777"
+setup = """
+
+This rule requires data coming in from Elastic Defend.
+
+### Elastic Defend Integration Setup
+Elastic Defend is integrated into the Elastic Agent using Fleet. Upon configuration, the integration allows
+the Elastic Agent to monitor events on your host and send data to the Elastic Security app.
+
+#### Prerequisite Requirements:
+- Fleet is required for Elastic Defend.
+- To configure Fleet Server refer to the [documentation](https://www.elastic.co/guide/en/fleet/current/fleet-server.html).
+
+#### The following steps should be executed in order to add the Elastic Defend integration on a Linux System:
```

Comment:
This is very nice and detailed, but I wonder about sustainability. With changes to the UI every minor, we may create unwanted debt by including this throughout the ruleset. We could always point to the docs sight for Defend and rely upstream on the integration owners to keep docs updated.

Maybe we good question for the next team or area meeting to discuss.

Reply:
This was added by @shashank-elastic 's PRs. There were several, but as an example, this one: https://github.com/elastic/detection-rules/pull/3141. It was approved by our content manager, so I am just adding it to the new ones.